examples/set-namespace/README.md

```diff
@@ -1,25 +0,0 @@
-# Set Namespace
```

Comment:
These files are used by e2e tests. if you delete them, you have to update the e2e tests to use new files.

Reply:
I guess you mean the stuff under `examples/set-namespace/configs/`.
IMO we need to overhaul our e2e tests anyway to make it look like what we desire. I guess it won't be a big problem. Please let me know if it is.
`mdrip --mode test --label testMaster README.md` should also be added somewhere as part of the presubmit. We need to figure it out.